towncrier.toml

```diff
@@ -0,0 +1,37 @@
+[tool.towncrier]
+directory = "changelog.d"
+filename = "CHANGELOG.md"
+start_string = "<!-- towncrier release notes start -->\n"
+title_format = "## [{version}](https://github.com/omnivector-solutions/jobbergate/releases/tag/{version}) - {project_date}"
+issue_format = "[ASP-{issue}](https://jiratoolchain.collaborationlayer-traton.com/browse/ASP-{issue})"
```

Comment:
This is up for discussion. What should we use by default? Jira, clickup, or PR numbers?

Reply:
Well, both ClickUp and JIRA do not have public urls (by default; we can manually create sharable urls from ClickUp), so I think neither will work.

Reply:
I changed to PR number to make it more general on the project for now. Nothing stops us from adding links manually using markdown syntax on news files when needed.